### PR TITLE
[docs] Improve the readme and issue template

### PR DESCRIPTION
**Pull request by leonardBang**
Improve the readme and issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-notice.yml [hunk 1]
```diff
@@ -47,6 +50,8 @@ body:
         感谢您选择 Flink CDC! 
 
         由于 Apache Flink 的要求，请在 [Apache Jira](https://issues.apache.org/jira) 的 `Flink` 项目下使用 `Flink CDC` 标签来反馈缺陷或新功能。
+        您需要登录 Jira 帐户才能创建 issue 去记录缺陷或新功能，如果您没有 Apache JIRA 帐户，您可以在 [ASF Jira 账号](https://selfserve.apache.org/jira-account.html)
+        申请账号，账号创建会由申请项目的 PMC member 审核，通常一到两个工作日可以通过。
```
Comment (caicancai) [inline .github/ISSUE_TEMPLATE/issue-notice.yml:54]: ASF Self-serve portal == ASF Jira 账号？

Comment (leonardBang) [reply .github/ISSUE_TEMPLATE/issue-notice.yml:54]: selfserve.apache.org/jira-account  == ASF Jira 账号

Comment (leonardBang) [reply .github/ISSUE_TEMPLATE/issue-notice.yml:54]: 本意是想照顾下中文用户，特别是新用户，使用 “ASF自助服务门户” 会让中文新用户特别懵，所以还是“ ASF Jira 账号” 用户好理解

File: .github/ISSUE_TEMPLATE/issue-notice.yml [hunk 2]
```diff
@@ -47,6 +50,8 @@ body:
         感谢您选择 Flink CDC! 
 
         由于 Apache Flink 的要求，请在 [Apache Jira](https://issues.apache.org/jira) 的 `Flink` 项目下使用 `Flink CDC` 标签来反馈缺陷或新功能。
+        您需要登录 Jira 帐户才能创建 issue 去记录缺陷或新功能，如果您没有 Apache JIRA 帐户，您可以在 [ASF Jira 账号](https://selfserve.apache.org/jira-account.html)
```
Comment (caicancai) [inline .github/ISSUE_TEMPLATE/issue-notice.yml:53]: Apache JIRA 帐户 -> Apache Jira 帐户

**Review by caicancai — APPROVED** [review]
overall LGTM.
I left a small comment

**Review by loserwang1024 — APPROVED** [review]
LGTM!

**Review by Jiabao-Sun — APPROVED** [review]
LGTM

**Review by PatrickRen — APPROVED** [review]
@leonardBang Thanks for the patch! LGTM. 

What about linking the PR to a Jira issue?

**Comment by leonardBang** [comment]
> What about linking the PR to a Jira issue?
That's better but this is a minor doc improvement, thus I didn't open a jira ticket.

**Pull request merged**